launch/launch.go

```diff
@@ -2,13 +2,19 @@ package launch
 
 import (
 	"fmt"
+	"net/url"
 	"os/exec"
+	"path"
 
 	"github.com/screwdriver-cd/sd-local/config"
 	"github.com/screwdriver-cd/sd-local/screwdriver"
 )
 
-var lookPath = exec.LookPath
+var (
+	lookPath = exec.LookPath
+	apiVersion = "v4"
+	storeVersion = "v1"
+)
```

Comment:
```suggestion
const (
	apiVersion = "v4"
	storeVersion = "v1"
)

var (
	lookPath = exec.LookPath
)
```
[nits] I think it is better to declare api and store version by const.